production/resources/js/sm-share-sidebar.js

```diff
@@ -0,0 +1,60 @@
+// Find and set page url, title, and summary
```

Comment:
So for now just leave these in. But moving forward please stay focused and make tighter pull requests.